### ISSUE TITLE
Doesn't work when imported into a web pack bundle.

### ISSUE THREAD

**Issue by masotime** [masotime/json-url#5]
Webpack is unable to deal with the implicit chunks in the `dist/browser` folder.

Possible solution - looking into using [pkg.module](https://github.com/rollup/rollup/wiki/pkg.module)

**Comment by sergiubologa** [comment]
Is there a workaround or some quick way to make it work, even if it's not the optimal solution?
@davidhpark did you find a quick fix?

**Comment by sergiubologa** [comment]
I am using `create-react-app` and I get the same issue. 

I managed to make it work by importing the other chunks also, like this:

```
import JsonUrl from 'json-url'
import 'json-url/dist/browser/json-url-msgpack'
import 'json-url/dist/browser/json-url-lzw'
import 'json-url/dist/browser/json-url-lzma'
import 'json-url/dist/browser/json-url-lzstring'
import 'json-url/dist/browser/json-url-safe64'
```

Hope this helps other people until a fix is implemented.

**Comment by masotime** [comment]
Thanks @sergiubologa ! Directly loading the chunks is a good workaround. I'll try and get a fix in soon.

**Comment by masotime** [comment]
Sorry for the really long delay - I've made the entry point for Webpack point to a single bundle instead of individual chunks - fix should be in 2.4.0, let me know if that works.

**Comment by masotime** [comment]
Closing for now. Please re-open if the issue still persists.
